app/assets/stylesheets/custom.scss

```diff
@@ -6,16 +6,38 @@
 //header logo
 #logo {
   float: left;
-  margin-right: 10px;
+  margin-left: 25px;
   font-size: 1.8em;
+  font-family: Arial;
   color: #FFD600;
   text-transform: capitalize;
-  letter-spacing: -1px;
-  padding-top: 5px;
+  letter-spacing: 0px;
+  padding-top: 8px;
   font-weight: 900 ;
   &:hover {
     color:deepskyblue;
     text-decoration: none;
   }
 }
 
+#login{
```

Comment:
create login.css.scss and put all css related things to this file
add a space before "{"